src/plugins/hidsim/hid_injection.cpp

```diff
@@ -220,12 +241,10 @@ static int get_display_dimensions(qmp_connection* qc, dimensions* dims)
     if (matches != 2 || dims->width < 0 || dims->height < 0)
```

Comment:
Code scanning

Comparison result is always the same
Comparison is always false because width >= 0.

Reply:
Changed and forcely pushed. Thanks for reviewing!